docs/USING_PUBLIC_CLOUDS.rst

```diff
@@ -0,0 +1,255 @@
+writing your data to. There are a few configurable options here:
+
+- :code:`type` : set to :code:`azure` to indicate this region is hosted in MS Azure;
+- :code:`legacyAwsBehavior` : set to :code:`false`;
+- :code:`azureBlobEndpoint` : set to your storage account's endpoint, usually
+  :code:`https://{{storageAccountName}}.blob.core.windows.name`;
+- :code:`azureContainerName` : set to an *existing container* in your MS Azure
+  storage account; this is the container that will host all the data written in
+  this region;
+- :code:`bucketMatch` : set to :code:`true` if you want your object name to be
+  the same in your local bucket and your MS Azure container; set to
+  :code:`false` if you want your object name to be of the form
+  :code:`{{localBucketName}}/{{objectname}}` in your MS Azure container
+- :code:`azureBlobSAS` : provide a generated Shared Access Signature *without*
+  the initial question mark;
+
+.. code:: json
+
+    (...)
+    "azure-test": {
+	"type": "azure",
+        "legacyAwsBehavior": false,
+        "details": {
+          "azureBlobEndpoint": "https://zenkomeetups.blob.core.windows.net/",
+	  "bucketMatch": true,
+	  "azureBlobSAS": "sv=2017-04-17&ss=b&srt=sco&sp=rwdlac&se=2017-04-17T00:13:19Z&st=2017-10-10T16:13:19Z&spr=https&sig=R%2FUNb%2BjVrKrOsHyr%2F8V%2FpD%2F18%2BgUeYopsHxH%2Fjre9XNc%3K",
+	  "azureContainerName": "zenkontainer"
+	}
+    },
+    (...)
+
+Start the server with the ability to write to MS Azure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Inside the repository, once all the files have been edited, you should be able
+to start the server and start testing pushing to MS Azure.
+
+.. code:: shell
+
+   # Start the server locally
+   $> S3DATA=multiple npm start
+
+Run the server as a docker container with the ability to write to MS Azure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mount all the files that have been edited to override defaults, and do a
+standard Docker run; then you can start testing pushing to MS Azure.
+
+.. code:: shell
+
+   # Start the server in a Docker container
+   $> sudo docker run -d --name CloudServer \
+   -v $(pwd)/data:/usr/src/app/localData \
+   -v $(pwd)/metadata:/usr/src/app/localMetadata \
+   -v $(pwd)/locationConfig.json:/usr/src/app/locationConfig.json \
+   -v $(pwd)/conf/authdata.json:/usr/src/app/conf/authdata.json \
+   -e S3DATA=multiple -e ENDPOINT=http://localhost -p 8000:8000
+   -d scality/s3server
+
+Testing: put an object to MS Azure using CloudServer
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to start testing pushing to MS Azure, you will need to create a local
+bucket in the MS Azure region - this local bucket will only store the metadata
+locally, while both the data and the metadata will be stored on MS Azure.
+This example is based on all our previous steps.
+
+.. code:: shell
+
+   # Create a local bucket hosting data in AWS S3
```

Comment:
'MS Azure' instead of 'AWS S3'? (also in line 236)

Reply:
Fixed, thanks

Reply:
storing data in AWS S3.  I don't like the term "hosted" because it is confusing with AWS's hosted style buckets.

Reply:
Changed both